tests/utest/zap/test_zap_cli.py

```diff
@@ -0,0 +1,118 @@
+from ..helpers import RESOURCES_PATH
+from oxygen.oxygen import OxygenCLI
+from robot.running.model import TestSuite
+from unittest import TestCase
+from unittest.mock import ANY, create_autospec, Mock, patch
+
+
+import sys
```

Comment:
The pep8-compliant ordering should be:
```suggestion
import sys
from unittest import TestCase
from unittest.mock import ANY, create_autospec, Mock, patch

from robot.running.model import TestSuite

from oxygen.oxygen import OxygenCLI
from ..helpers import RESOURCES_PATH
```
To explain:
```python
# first come imports from standard library
# and first of them, imports of full modules
import sys  
# and then "from ... import"s
from unittest import TestCase  
from unittest.mock import ANY, create_autospec, Mock, patch

# we also leave one empty line to denote to others that "section" of imports has changed

# then come imports from 3rd party modules
from robot.running.model import TestSuite

# another empty line to denote "section"

# then come imports from local code
from oxygen.oxygen import OxygenCLI
# with relative imports being last because they are the 'closest' 
from ..helpers import RESOURCES_PATH